src/frontend/screens/Onboarding/OnboardingPrivacyPolicy.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import {
+  View,
+  Text,
+  ScrollView,
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native';
+import {useIntl, defineMessages} from 'react-intl';
+import {PrivacyPolicy} from '../PrivacyPolicy';
+import {BLUE_GREY, WHITE, BLACK, COMAPEO_BLUE} from '../../lib/styles';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+
+import {AppDiagnosticMetrics} from '../../metrics/AppDiagnosticMetrics';
+import {DeviceDiagnosticMetrics} from '../../metrics/DeviceDiagnosticMetrics';
+
+const m = defineMessages({
+  navTitle: {
+    id: 'screens.OnboardingPrivacyPolicy.navTitle',
+    defaultMessage: 'Privacy Policy',
+  },
+  permissionsTitle: {
+    id: 'screens.OnboardingPrivacyPolicy.permissionsTitle',
+    defaultMessage: 'Current Permissions',
+  },
+  shareDiagnostics: {
+    id: 'screens.OnboardingPrivacyPolicy.shareDiagnostics',
+    defaultMessage: 'Share Diagnostic Information',
+  },
+});
+
+export const OnboardingPrivacyPolicy = () => {
+  const appDiagnosticMetrics = new AppDiagnosticMetrics();
+  const deviceDiagnosticMetrics = new DeviceDiagnosticMetrics();
+  const {formatMessage} = useIntl();
+  const [isDiagnosticsEnabled, setIsDiagnosticsEnabled] = React.useState(false);
+
+  const toggleDiagnostics = () => {
+    setIsDiagnosticsEnabled(prev => !prev);
+  };
+
+  React.useEffect(() => {
+    appDiagnosticMetrics.setEnabled(isDiagnosticsEnabled);
+    deviceDiagnosticMetrics.setEnabled(isDiagnosticsEnabled);
+  }, [isDiagnosticsEnabled]);
+
+  return (
+    <ScrollView contentContainerStyle={styles.scrollContent}>
+      <PrivacyPolicy />
+      <View style={styles.horizontalLine} />
+      <Text style={styles.header}>{formatMessage(m.permissionsTitle)}</Text>
+      <View style={styles.permissionBox}>
+        <Text style={styles.permissionText}>
+          {formatMessage(m.shareDiagnostics)}
+        </Text>
+        <TouchableOpacity
+          onPress={toggleDiagnostics}
+          style={[
+            styles.checkBox,
+            isDiagnosticsEnabled && styles.checkBoxChecked,
+          ]}>
+          {isDiagnosticsEnabled && (
+            <MaterialIcons name="check" size={18} color={WHITE} />
+          )}
+        </TouchableOpacity>
```

Comment:
this is pretty difficult to press. would suggest either:

1. asking design whether it would make sense to make the whole row pressable
2. increasing the hit slop of this